napari/utils/shortcuts.py

```diff
@@ -43,5 +45,5 @@
     'napari:delete_selected_shapes': ['Backspace', 'Delete', '3'],
     'napari:finish_drawing_shape': ['Escape'],
     'napari:reset_active_layer_affine': ['Control-Shift-R'],
-    'napari:transform_active_layer': ['Control-Shift-A'],
+    'napari:transform_active_layer': ['Meta-Shift-A', 'Alt-Shift-A'],
```

Comment:
See main comments about Meta/Alt being actually used to write on some language/keyboard. 
I get we are on a context where users may rarely write, and I would have bigger concern with some other combinaisons of `Atl-smth`, so that's not a blocker for me.